src/net.cpp

```diff
@@ -1641,6 +1641,7 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
     auto next_extra_block_relay = GetExponentialRand(start, EXTRA_BLOCK_RELAY_ONLY_PEER_INTERVAL);
     const bool dnsseed = gArgs.GetBoolArg("-dnsseed", DEFAULT_DNSSEED);
     bool add_fixed_seeds = gArgs.GetBoolArg("-fixedseeds", DEFAULT_FIXEDSEEDS);
+    const bool use_seednodes{gArgs.IsArgSet("-seednode")};
```

Comment:
I think I'd prefer to leave `add_fixed_seeds_now` within the current scope, especially because my plan is to follow up by moving the entire fixed seed logic out of `ThreadOpenConnections` into what is currently `ThreadDNSAddressSeed` (currently #26114). One reason for opening this PR is so that the move into another thread can be a refactor and doesn't change timing behavior.